validation/policies/io/konveyor/forklift/ovirt/tpm.rego

```diff
@@ -0,0 +1,16 @@
+package io.konveyor.forklift.ovirt
+
+default has_tpm_enabled = false
+
+has_tpm_enabled = value {
+   value :=  input.tpmEnabled
+}
+
+concerns[flag] {
+    has_tpm_enabled
+    flag := {
+        "category": "Warning",
+        "label": "VM configured with TPM device",
+        "assessment": "The VM is configured with TPM device. TPM data will be loss during the migration."
```

Comment:
```suggestion
        "assessment": "The VM is configured with a TPM device. TPM data is not transferred during the migration."
```
("will be loss" may sound like it's removed from the source, so let's be more explicit about what would happen)